[2025-01-31T04:47:02.172+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag1_v2.second_task scheduled__2022-09-19T00:00:00+00:00 [queued]>
[2025-01-31T04:47:02.185+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag1_v2.second_task scheduled__2022-09-19T00:00:00+00:00 [queued]>
[2025-01-31T04:47:02.186+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2025-01-31T04:47:02.210+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): second_task> on 2022-09-19 00:00:00+00:00
[2025-01-31T04:47:02.219+0000] {standard_task_runner.py:57} INFO - Started process 4475 to run task
[2025-01-31T04:47:02.224+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag1_v2', 'second_task', 'scheduled__2022-09-19T00:00:00+00:00', '--job-id', '1233', '--raw', '--subdir', 'DAGS_FOLDER/dag_example.py', '--cfg-path', '/tmp/tmpozkd2upg']
[2025-01-31T04:47:02.225+0000] {standard_task_runner.py:85} INFO - Job 1233: Subtask second_task
[2025-01-31T04:47:02.355+0000] {task_command.py:415} INFO - Running <TaskInstance: dag1_v2.second_task scheduled__2022-09-19T00:00:00+00:00 [running]> on host f8bdfcb8bca5
[2025-01-31T04:47:02.532+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='anhtupham99' AIRFLOW_CTX_DAG_ID='dag1_v2' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2022-09-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-09-19T00:00:00+00:00'
[2025-01-31T04:47:02.534+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-01-31T04:47:02.535+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo I am the second task, I run after the first task']
[2025-01-31T04:47:02.555+0000] {subprocess.py:86} INFO - Output:
[2025-01-31T04:47:02.557+0000] {subprocess.py:93} INFO - I am the second task, I run after the first task
[2025-01-31T04:47:02.558+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-01-31T04:47:02.593+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag1_v2, task_id=second_task, execution_date=20220919T000000, start_date=20250131T044702, end_date=20250131T044702
[2025-01-31T04:47:02.641+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-01-31T04:47:02.662+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
